.github/workflows/node.test.yaml

```diff
@@ -90,3 +90,4 @@ jobs:
           DATABASE_URL: "postgresql://postgres:postgres@localhost:5432/orakl-test?search_path=public"
```

Comment:
I think this line contains the user name and password. Although it's just a local test, good practice is to move it to secret

Reply:
it's also written on setting up pgsql for github action, starting from line 16. it's unnecessary to hide connection info for temporary db created and removed in github action.